nbactions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we can remove this file now.

Reply:
Yes sure, will do (sorry that is my IDE)